src/tailwind.js

```diff
@@ -1,86 +1,74 @@
-module.exports = {
+const _ = require("lodash"); //eslint-disable-line @typescript-eslint/no-var-requires
+const commonUiConfig = require("@govtechsg/tradetrust-ui-components/build/tailwind"); //eslint-disable-line @typescript-eslint/no-var-requires
+
+// https://tailwindcss.com/docs/theme
+const localConfig = {
   // future: {
   //   removeDeprecatedGapUtilities: true,
   //   purgeLayersByDefault: true,
   // },
   // purge: ["./src/**/*.ts", "./src/**/*.tsx"], // not to purge, cos common ui might need certain styles (should purge specific layers instead if really need)
   theme: {
```

Comment:
purge the css?

Reply:
oh ya, ok i'll add what u did for creator into here